### ISSUE TITLE
Encounter Operation

### ISSUE THREAD

**Issue by OPAnimeMC** [pipirapira/triggernometry-pro#4]
I was wondering if you know if there is anyway to keep an encounter going when you start it using the start encounter action until you send another trigger to end it.

**Comment by pipirapira** [comment]
ACT will automatically start and keep an encounter when it receives an event containing combat information.
The "start encounter" action will send a combat event similar to other events generated by the FFXIV_ACT_Plugin.
If FFXIV_ACT_Plugin sends a combat event, current encounter will be extended, if not, you can use that action again.

**Comment by OPAnimeMC** [comment]
Yes that's how i figured it would work, but thats not how is happening for me, whenever i fire a trigger with start encounter it creates a new encounter with all the info like its supposed to but its ened immediatly

**Comment by OPAnimeMC** [comment]
Ok, i think i got it to work. I tried turning off the setting in Main table - General, the number of seconds to wait after last combat action to begin new encounter, thank you again.
